README.md

```diff
@@ -6,15 +6,22 @@ Clean, Uniform and Refined with Automatic Tracking from Experimental Database (C
 This repository contains original structures from literature search, with git-tracked corrections. \
 DFT-optimized frameworks with DDEC charges are available on [Materials Cloud](https://www.materialscloud.org/discover/curated-cofs).
 
+* `cof-frameworks.csv` contains the list of COFs
+* `cof-discarded.csv` contains a list of COFs that have been discarded because of known duplicate, discarded layering, discarded n-folding, or other reasons reported in the document
```

Comment:
Does `n-folding` refer to interpenetration?

```suggestion
* `cof-discarded.csv` contains a list of COFs that have been discarded because of duplicate detection, unobserved stacking variant, unobserved interpenetration, or other reasons reported in the document
```

Reply:
modified